--- gupnp-tools-0.8.12/data/gupnp-av-cp.desktop.in.in.orig	2016-05-11 22:13:19.590079996 +0200
+++ gupnp-tools-0.8.12/data/gupnp-av-cp.desktop.in.in	2016-05-11 22:15:54.713406820 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=UPnP AV Control Point
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-av-cp
 Icon=@PKGDATADIR@/pixmaps/av-cp.png
 Terminal=false
--- gupnp-tools-0.8.12/data/gupnp-network-light.desktop.in.in.orig	2016-05-11 22:13:19.590079996 +0200
+++ gupnp-tools-0.8.12/data/gupnp-network-light.desktop.in.in	2016-05-11 22:16:42.286738157 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=UPnP Network Light
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-network-light
 Icon=@PKGDATADIR@/pixmaps/network-light-256x256.png
 Terminal=false
--- gupnp-tools-0.8.12/data/gupnp-universal-cp.desktop.in.in.orig	2016-05-11 22:13:19.590079996 +0200
+++ gupnp-tools-0.8.12/data/gupnp-universal-cp.desktop.in.in	2016-05-11 22:17:11.430070272 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=UPnP Universal Control Point
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-universal-cp
 Icon=@PKGDATADIR@/pixmaps/universal-cp.png
 Terminal=false
