--- gupnp-tools-0.8.15/data/gupnp-av-cp.desktop.in.orig	2018-12-10 17:34:41.475536944 +0100
+++ gupnp-tools-0.8.15/data/gupnp-av-cp.desktop.in	2018-12-10 17:36:25.845535752 +0100
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=UPnP AV Control Point
 Comment=Browse and control UPnP multimedia devices
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-av-cp
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=av-cp
--- gupnp-tools-0.8.15/data/gupnp-network-light.desktop.in.orig	2018-12-10 17:34:41.475536944 +0100
+++ gupnp-tools-0.8.15/data/gupnp-network-light.desktop.in	2018-12-10 17:36:36.782202295 +0100
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=UPnP Network Light
 Comment=Sample UPnP application for controlling lights
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-network-light
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=network-light-256x256
--- gupnp-tools-0.8.15/data/gupnp-universal-cp.desktop.in.orig	2018-12-10 17:34:41.475536944 +0100
+++ gupnp-tools-0.8.15/data/gupnp-universal-cp.desktop.in	2018-12-10 17:36:47.982202167 +0100
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=UPnP Universal Control Point
 Comment=Inspect and control UPnP devices across your local network
-Categories=GTK;Development;
+Categories=GTK;Network;RemoteAccess;
 Exec=gupnp-universal-cp
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=universal-cp
