--- gupnp-tools-0.10.1/src/universal-cp/main.c.orig	2021-06-21 23:23:20.515533000 +0200
+++ gupnp-tools-0.10.1/src/universal-cp/main.c	2021-06-22 20:30:04.384036023 +0200
@@ -74,7 +74,7 @@
                 g_set_error (error,
                              G_IO_ERROR,
                              G_IO_ERROR_INVALID_ARGUMENT,
-                             msg);
+                             "%s", msg);
                 g_free (msg);
 
                 return FALSE;
