--- gupnp-tools-0.10.0/src/common/meson.build.orig	2019-03-29 20:09:19.000000000 +0100
+++ gupnp-tools-0.10.0/src/common/meson.build	2019-04-17 17:14:21.022029404 +0200
@@ -1,7 +1,8 @@
 util_resource = gnome.compile_resources('common-resource',
     join_paths(resource_data, 'org.gupnp.Tools.Common.gresource.xml'),
     source_dir : resource_data,
-    extra_args : ['--manual-register', '--c-name', 'gupnp_tools_common']
+    extra_args : ['--manual-register'],
+    c_name: 'gupnp_tools_common'
 )
 
 libutil = static_library('util', ['icons.c', 'icons.h', 'pretty-print.c', 'pretty-print.h'] + util_resource,
