--- gupnp-tools-0.8.5/configure.ac.orig	2013-01-08 17:41:31.097483890 +0100
+++ gupnp-tools-0.8.5/configure.ac	2013-01-08 17:42:14.607482983 +0100
@@ -52,7 +52,6 @@
 PKG_CHECK_MODULES(LIBXML, libxml-2.0 >= LIBXML_REQUIRED)
 PKG_CHECK_MODULES(GTK, gtk+-3.0 >= $GTK_REQUIRED)
 PKG_CHECK_MODULES(GTHREAD, gthread-2.0 >= $GTHREAD_REQUIRED)
-PKG_CHECK_MODULES(GNOME_ICON_THEME, gnome-icon-theme >= $ICON_THEME_REQUIRED)
 PKG_CHECK_MODULES(GIO, glib-2.0 >= $GLIB_REQUIRED
                        gobject-2.0 >= $GLIB_REQUIRED
                        gio-2.0 >= $GLIB_REQUIRED)
